megamek/src/megamek/common/MechSummaryCache.java

```diff
@@ -142,7 +142,7 @@ public void removeListener(Listener listener) {
     private MechSummaryCache() {
         m_nameMap = new HashMap<>();
         m_fileNameMap = new HashMap<>();
-        LOG.setLogLevel(this, LOGLVL);
+        MegaMek.getLogger().setLogLevel(LOGLVL);
```

Comment:
Same as before, is this setting just for this class or in general, and why is is hardcoded.